PHP code style check is checking data folder and apps2 and apps3 folders

### Steps to reproduce
1. install development ownCloud in local git repo with the data folder in ``data`` of the local repo
2. as some user, upload some *.php code with crap style
3. ``make test-php-style``

### Expected behaviour
Style checks pass

### Actual behaviour
Style check fails on the file(s) that are actually just test user data, not real ownCloud code

### Server configuration
Local development environment.

I'm reopening this because even with `.php_cs` it doesn't work for me.

I tried dumping the array:
```
lib/composer/bin/php-cs-fixer fix -v --diff --diff-format udiff --dry-run --allow-risky yes
array(12) {
  [0]=>
  string(12) "lib/composer"
  [1]=>
  string(5) "build"
  [2]=>
  string(28) "apps/files_external/3rdparty"
  [3]=>
  string(6) "config"
  [4]=>
  string(4) "data"
  [5]=>
  string(5) "apps2"
  [6]=>
  string(5) "apps3"
  [7]=>
  string(16) "apps/objectstore"
  [8]=>
  string(18) "apps/theme-thunder"
  [9]=>
  string(15) "apps/theme-fire"
  [10]=>
  string(17) "apps/multidirtest"
  [11]=>
  string(13) "apps/contacts"
}
Loaded config ownCloud coding standard from "/srv/www/htdocs/owncloud/.php_cs".
FFFF.......................................................................................................................................................................................................................................................................................................................................................^Cmake: *** [Makefile:194: test-php-style] Interrupt
```

Since I run into an out of memory error even with 2GB, I cannot see where the errors are pointing to so will need to do a trial and error session to find out.

Maybe we should rather use inclusion instead of exclusion ?

This is a blocker for using git hooks https://github.com/owncloud/core/pull/31837

--- a/.gitignore
+++ b/.gitignore
@@ -127,3 +127,4 @@ Vagrantfile
 clover.xml
 /tests/output
 .php_cs.cache
+.php_cs

--- a/.php_cs.dist
+++ b/.php_cs.dist
@@ -4,41 +4,41 @@ $dirToParse = 'apps';
 $dirIterator = new DirectoryIterator(__DIR__ . '/' . $dirToParse);
 
 $bundledApps = [
-    'comments',
-    'dav',
-    'federatedfilesharing',
-    'federation',
-    'files',
-    'files_external',
-    'files_sharing',
-    'files_trashbin',
-    'files_versions',
-    'provisioning_api',
-    'systemtags',
-    'testing',
-    'updatenotification'
+	'comments',
+	'dav',
+	'federatedfilesharing',
+	'federation',
+	'files',
+	'files_external',
+	'files_sharing',
+	'files_trashbin',
+	'files_versions',
+	'provisioning_api',
+	'systemtags',
+	'testing',
+	'updatenotification'
 ];
 
 $excludeDirs = [
-    'lib/composer',
-    'build',
-    'apps/files_external/3rdparty',
-    'config',
-    'data',
-    '3rdparty',
+	'lib/composer',
+	'build',
+	'apps/files_external/3rdparty',
+	'config',
+	'data',
+	'3rdparty',
 ];
 
 foreach ($dirIterator as $fileinfo) {
-    $filename = $fileinfo->getFilename();
-    if ($fileinfo->isDir() && !$fileinfo->isDot() && !in_array($filename, $bundledApps)) {
-        $excludeDirs[] = $dirToParse . '/' . $filename;
-    }
+	$filename = $fileinfo->getFilename();
+	if ($fileinfo->isDir() && !$fileinfo->isDot() && !in_array($filename, $bundledApps)) {
+		$excludeDirs[] = $dirToParse . '/' . $filename;
+	}
 }
 
 $finder = PhpCsFixer\Finder::create()
-    ->exclude($excludeDirs)
-    ->notPath('tests/autoconfig*')
-    ->in(__DIR__);
+	->exclude($excludeDirs)
+	->notPath('tests/autoconfig*')
+	->in(__DIR__);
 
 $config = new OC\CodingStandard\Config();
 $config->setFinder($finder);

--- a/.php_cs.dist
+++ b/.php_cs.dist
@@ -24,7 +24,8 @@ $excludeDirs = [
     'build',
     'apps/files_external/3rdparty',
     'config',
-    'data'
+    'data',
+    '3rdparty',
 ];
 
 foreach ($dirIterator as $fileinfo) {
@@ -36,6 +37,7 @@ foreach ($dirIterator as $fileinfo) {
 
 $finder = PhpCsFixer\Finder::create()
     ->exclude($excludeDirs)
+    ->notPath('tests/autoconfig*')
     ->in(__DIR__);
 
 $config = new OC\CodingStandard\Config();

--- a/.php_cs.dist
+++ b/.php_cs.dist
@@ -23,6 +23,7 @@ $excludeDirs = [
 	'lib/composer',
 	'build',
 	'apps/files_external/3rdparty',
+	'apps-external',
 	'config',
 	'data',
 	'3rdparty',

